lib/livebook/hubs.ex

```diff
@@ -84,7 +88,7 @@ defmodule Livebook.Hubs do
       end
 
       :ok = Storage.delete(@namespace, id)
-      :ok = broadcast_hubs_change()
+      :ok = broadcast_message(:crud, {:hubs_metadata_changed, get_metadatas()})
```

Comment:
Let's just broadcast it as `:hubs_metadata_changed` without the full list for consistency?